go.mod

```diff
@@ -3,7 +3,53 @@ module gopkg.in/DataDog/dd-trace-go.v1
 go 1.12
 
 require (
+	cloud.google.com/go/pubsub v1.10.1
 	github.com/DataDog/datadog-go v4.4.0+incompatible
+	github.com/Shopify/sarama v1.28.0
+	github.com/aws/aws-sdk-go v1.38.16
+	github.com/bradfitz/gomemcache v0.0.0-20190913173617-a41fca850d0b
+	github.com/confluentinc/confluent-kafka-go v1.6.1
+	github.com/emicklei/go-restful v2.15.0+incompatible
+	github.com/garyburd/redigo v1.6.2
+	github.com/gin-gonic/gin v1.7.1
+	github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8
+	github.com/go-chi/chi v1.5.4
+	github.com/go-chi/chi/v5 v5.0.2
+	github.com/go-pg/pg/v10 v10.9.0
+	github.com/go-redis/redis v6.15.9+incompatible
+	github.com/go-redis/redis/v7 v7.4.0
+	github.com/go-redis/redis/v8 v8.8.0
+	github.com/gocql/gocql v0.0.0-20210401103645-80ab1e13e309
+	github.com/gofiber/fiber/v2 v2.7.1
+	github.com/golang/protobuf v1.5.2
+	github.com/gomodule/redigo v1.8.4
 	github.com/google/pprof v0.0.0-20210125172800-10e9aeb4a998
+	github.com/google/uuid v1.2.0
+	github.com/gorilla/mux v1.8.0
+	github.com/graph-gophers/graphql-go v0.0.0-20210329110328-21b77bdd71c6
+	github.com/hashicorp/consul/api v1.8.1
+	github.com/hashicorp/vault/api v1.1.0
+	github.com/hashicorp/vault/sdk v0.2.0
+	github.com/jinzhu/gorm v1.9.16
+	github.com/jmoiron/sqlx v1.3.3
+	github.com/julienschmidt/httprouter v1.3.0
+	github.com/labstack/echo v3.3.10+incompatible
+	github.com/labstack/echo/v4 v4.2.2
+	github.com/miekg/dns v1.1.41
+	github.com/opentracing/opentracing-go v1.2.0
+	github.com/philhofer/fwd v1.1.1 // indirect
+	github.com/stretchr/testify v1.7.0
+	github.com/syndtr/goleveldb v1.0.0
+	github.com/tidwall/buntdb v1.2.3
 	github.com/tinylib/msgp v1.1.2
+	github.com/twitchtv/twirp v7.1.1+incompatible
+	github.com/zenazn/goji v1.0.1
+	go.mongodb.org/mongo-driver v1.5.1
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4
+	golang.org/x/oauth2 v0.0.0-20210402161424-2e8d93401602
+	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
+	google.golang.org/grpc v1.37.0
+	gopkg.in/jinzhu/gorm.v1 v1.9.2
+	gorm.io/gorm v1.21.7
 )
```

Comment:
this is contrib package, so it must not defend on too much other library

Reply:
I just ran `go mod tidy`. Will try to rollback changes in go.mod and go.sum